src/components/dialogs/PaymentErrorDialog.vue

```diff
@@ -0,0 +1,57 @@
+<template>
```

Comment:
Basically the same component as in Filings UI. (I use an Unordered List for the PayBC times as that looked better.)